src/sentry/conf/urls.py

```diff
@@ -28,9 +28,12 @@
 
 urlpatterns = patterns(
     '',
-    url(r'^admin/', include(django_admin.site.urls)),
     url(r'^500/', handler500, name='error-500'),
     url(r'^404/', handler404, name='error-404'),
     url(r'^403-csrf-failure/', CsrfFailureView.as_view(), name='error-403-csrf-failure'),
     url(r'^__debug__/', include(debug_toolbar.urls)),
 ) + web_urlpatterns
+
+if 'django.contrib.admin' in settings.INSTALLED_APPS:
+    from sentry import django_admin
+    urlpatterns += url(r'^admin/', include(django_admin.site.urls)),
```

Comment:
I don't think this will work unless it's before `web_urlpatterns.

I think you'd wanna do like:

```python
urlpatterns = patterns(
    '',
    url(r'^500/', handler500, name='error-500'),
    url(r'^404/', handler404, name='error-404'),
    url(r'^403-csrf-failure/', CsrfFailureView.as_view(), name='error-403-csrf-failure'),
    url(r'^__debug__/', include(debug_toolbar.urls)),
)

if 'django.contrib.admin' in settings.INSTALLED_APPS:
    from sentry import django_admin
    urlpatterns += url(r'^admin/', include(django_admin.site.urls))

urlpatterns += web_urlpatterns
```

`web_urlpatterns` ends with a catch all to dump into React.

You also have a mistake on this line with a traling comma that would I think probably crash things.

Reply:
Oh, jk, the trailing comma is intentional here I guess since you're appending a tuple to a tuple. Ugh, python.